cachito/web/models.py

```diff
@@ -162,6 +175,18 @@ def number_of_deps(self):
         return db.session.query(sqlalchemy.func.count(req_dep_cols.dependency_id)).filter(
             req_dep_cols.request_id == self.id).scalar()
 
+    @property
+    def number_of_dep_overrides(self):
```

Comment:
How about renaming this to dependency_overrides_count

Reply:
@yashvardhannanavati I'm just using that name to be consistent with `number_of_deps`. What's funny is you named it that way in #52.  :laughing:

If I rename this one, I'll also rename the other. Which do you prefer?

Reply:
@mprahl lol, Sorry I didn't realize it while writing #52. Can I please request you to rename both if possible?